controlpanel/api/models/feedback.py

```diff
@@ -0,0 +1,25 @@
+# Third-party
+from django.db import models
+from django.utils import timezone
+
+
+class Feedback(models.Model):
+    SATISFACTION_RATINGS = [
+        (5, "Very satisfied"),
+        (4, "Satisfied"),
+        (3, "Neither satisfied or dissatisfied"),
+        (2, "Dissatisfied"),
+        (1, "Very dissatisfied"),
+    ]
+
+    satisfaction_rating = models.IntegerField(
+        choices=SATISFACTION_RATINGS,
+        null=False,
+        blank=False,
+    )
+
+    suggestions = models.TextField()
```

Comment:
I wonder if we want to make this optional? Not sure either way, so maybe check with the team - but if so should just be a case of adding `blank=True`